ert storage test workspace integration times out (flaky)

The CI in https://github.com/equinor/ert/pull/2398 runs the following test: https://ci.equinor.com/scout/job/komodo_testing/job/ert-custom-script-pr/2551

which fails in `workspace_integration`

```
ert3/engine/integration/test_engine.py ...............E                  [ 71%]

==================================== ERRORS ====================================
_________ ERROR at setup of test_incompatible_partial_sensitivity_run __________

tmpdir = local('/tmp/pytest-of-f_scout_ci/pytest-841/test_incompatible_partial_sens0')

    @pytest.fixture()
    def workspace_integration(tmpdir):
        from ert_shared.services import Storage
    
        workspace_dir = pathlib.Path(tmpdir / "polynomial")
        workspace_dir.mkdir()
        os.chdir(workspace_dir)
    
        with Storage.start_server():
            workspace_obj = ert3.workspace.initialize(workspace_dir)
>           ert.storage.init(workspace_name=workspace_obj.name)

/tmp/jenkins-scout-f_scout_ci/workspace/komodo_testing/ert-custom-script-pr/test-root/tests/ert_tests/ert3/conftest.py:138: 

...

self = <ert_shared.services.storage_service.Storage object at 0x7f1de0ae03c8>
timeout = None

    def fetch_conn_info(self, timeout: Optional[float] = None) -> Mapping[str, Any]:
        self.wait_until_ready(timeout)
        if self._conn_info is None:
            raise ValueError("conn_info is None")
        elif isinstance(self._conn_info, Exception):
>           raise self._conn_info
E           TimeoutError

/tmp/jenkins-scout-f_scout_ci/workspace/komodo_testing/ert-custom-script-pr/test-kenv/root/lib64/python3.6/site-packages/ert_shared/services/_base_service.py:358: TimeoutError
```

likely related to https://github.com/equinor/ert/issues/2381

Adding that this also occurs in:
`test_sensitivity_fast_run_and_export`

I'm adding some more information to this one, as a second error message appeared for this test setup (and failed in combination with this)

```
__________ ERROR at setup of test_partial_sensitivity_run_and_export ___________
...
    def fetch_conn_info(self, timeout: Optional[float] = None) -> Mapping[str, Any]:
        self.wait_until_ready(timeout)
        if self._conn_info is None:
            raise ValueError("conn_info is None")
        elif isinstance(self._conn_info, Exception):
>           raise self._conn_info
E           ert_shared.services._base_service.ServerBootFail

/prog/res/komodo/2021.12.rc4-py36-rhel7/root/lib64/python3.6/site-packages/ert_shared/services/_base_service.py:359: ServerBootFail
...
OpenAPI Docs: http://st-linapp1194.st.statoil.no:51830/docs
INFO:     Started server process [22958]
INFO:uvicorn.error:Started server process [22958]
INFO:     Waiting for application startup.
INFO:uvicorn.error:Waiting for application startup.
INFO:     Application startup complete.
INFO:uvicorn.error:Application startup complete.
Traceback (most recent call last):
  File "/prog/res/komodo/2021.12.rc4-py36-rhel7/root/lib64/python3.6/site-packages/ert_shared/services/_storage_main.py", line 187, in <module>
    run_server(debug=False)
  File "/prog/res/komodo/2021.12.rc4-py36-rhel7/root/lib64/python3.6/site-packages/ert_shared/services/_storage_main.py", line 152, in run_server
    server.run(sockets=[sock])
  File "/prog/res/komodo/2021.12.rc4-py36-rhel7/root/lib/python3.6/site-packages/uvicorn/server.py", line 69, in run
    return asyncio.get_event_loop().run_until_complete(self.serve(sockets=sockets))
  File "/usr/lib64/python3.6/asyncio/base_events.py", line 484, in run_until_complete
    return future.result()
  File "/prog/res/komodo/2021.12.rc4-py36-rhel7/root/lib/python3.6/site-packages/uvicorn/server.py", line 86, in serve
    await self.startup(sockets=sockets)
  File "/prog/res/komodo/2021.12.rc4-py36-rhel7/root/lib64/python3.6/site-packages/ert_shared/services/_storage_main.py", line 24, in startup
    await super().startup(sockets)
  File "/prog/res/komodo/2021.12.rc4-py36-rhel7/root/lib/python3.6/site-packages/uvicorn/server.py", line 128, in startup
    handler, sock=sock, ssl=config.ssl, backlog=config.backlog
  File "/usr/lib64/python3.6/asyncio/streams.py", line 119, in start_server
    return (yield from loop.create_server(factory, host, port, **kwds))
  File "/usr/lib64/python3.6/asyncio/base_events.py", line 1082, in create_server
    sock.listen(backlog)
OSError: [Errno 98] Address already in use
```

--- a/tests/ert_tests/ert3/conftest.py
+++ b/tests/ert_tests/ert3/conftest.py
@@ -135,7 +135,7 @@ def workspace_integration(tmpdir):
     workspace_dir.mkdir()
     with chdir(workspace_dir):
 
-        with Storage.start_server():
+        with Storage.start_server(timeout=120):
             workspace_obj = ert3.workspace.initialize(workspace_dir)
             ert.storage.init(workspace_name=workspace_obj.name)
             yield workspace_obj

--- a/tests/ert_tests/ert3/console/integration/test_cli.py
+++ b/tests/ert_tests/ert3/console/integration/test_cli.py
@@ -601,7 +601,7 @@ def test_check_service(tmpdir, monkeypatch):
         with patch.object(
             sys, "argv", ["ert3", "service", "check", "storage", "--timeout", "10"]
         ):
-            with Storage.start_server():
+            with Storage.start_server(timeout=120):
                 ert3.console._console._main()
 
 

